pkgs/development/web/netlify-cli/generate.sh

```diff
@@ -2,13 +2,13 @@
 set -eu -o pipefail
 cd "$( dirname "${BASH_SOURCE[0]}" )"
 rm -f ./node-env.nix
-src="$(nix-build --expr '(import ../../../.. {}).fetchFromGitHub (lib.importJSON ./netlify-cli.json)')"
+src="$(nix-build --expr 'let pkgs = import ../../../.. {}; lib = import ../../../../lib; meta = (lib.importJSON ./netlify-cli.json); in pkgs.fetchFromGitHub {owner = meta.owner; repo = meta.repo; rev = meta.rev; sha256 = meta.sha256;}')"
```

Comment:
You can use `pkgs.lib` instead of a separate import. And `inherit (meta) owner repo rev sha256;`.

Reply:
okay, i'll change accordingly